sdk/python/touca/cli/_post.py

```diff
@@ -63,7 +67,7 @@ def run(self):
 
         api_key = options.get("api-key")
         api_url = options.get("api-url")
-
+        dry_run = self.__options.get("dry-run")
```

Comment:
Wouldn't it be better to use `options` instead of `self.__options`? I think all we'd need to do is to add `dry-run` to the list in line 58.

Reply:
Fixed. Not sure how I missed this but good catch